website/content/en/preview/troubleshooting.md

```diff
@@ -314,6 +314,33 @@ When a node is launched by Karpenter, it is assigned to a subnet within your VPC
 
 For more troubleshooting information on why your pod may have a `FailedCreateSandbox` error, view the [EKS CreatePodSandbox Knowledge Center Post](https://repost.aws/knowledge-center/eks-failed-create-pod-sandbox).
 
+### Windows pods are failing with `FailedCreatedPodSandbox`
+
+The following solution(s) may resolve your issue if you are seeing an error similar to the following when attempting to launch Windows pods. This error typically occurs if you have not enabled Windows support.
+
+```
+Failed to create pod sandbox: rpc error: code = Unknown desc = failed to setup network for sandbox "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx": plugin type="vpc-bridge" name="vpc" failed (add): failed to parse Kubernetes args: pod does not have label vpc.amazonaws.com/PrivateIPv4Address
+```
+
+#### Solutions
+
+1. See documentation [here](https://docs.aws.amazon.com/eks/latest/userguide/windows-support.html#enable-windows-support) for how to enable Windows support.
```

Comment:
I'd write:
```suggestion
1. See [Enabling Windows support](https://docs.aws.amazon.com/eks/latest/userguide/windows-support.html#enable-windows-support) to learn more.
````

(that's roughly following a version of the Kubernetes style guide BTW)